_posts/0000-01-02-rubenoliveros.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
They said "Replace line 5 of your file with a quotation or meme and witty caption" and that's what I did

Reply:
Good boy!